price-feeder/oracle/provider/binance.go

```diff
@@ -163,56 +179,7 @@ func (p *BinanceProvider) GetCandlePrices(pairs ...types.CurrencyPair) (map[stri
 
 // SubscribeCurrencyPairs subscribe all currency pairs into ticker and candle channels.
 func (p *BinanceProvider) SubscribeCurrencyPairs(cps ...types.CurrencyPair) error {
-	if len(cps) == 0 {
-		return fmt.Errorf("currency pairs is empty")
-	}
-
-	if err := p.subscribeChannels(cps...); err != nil {
-		return err
-	}
-
-	p.setSubscribedPairs(cps...)
-	telemetryWebsocketSubscribeCurrencyPairs(ProviderBinance, len(cps))
-	return nil
-}
-
-// subscribeChannels subscribe to the ticker and candle channels for all currency pairs.
-func (p *BinanceProvider) subscribeChannels(cps ...types.CurrencyPair) error {
-	if err := p.subscribeTickers(cps...); err != nil {
-		return err
-	}
-
-	return p.subscribeCandles(cps...)
-}
-
-// subscribeTickers subscribe to the ticker channel for all currency pairs.
-func (p *BinanceProvider) subscribeTickers(cps ...types.CurrencyPair) error {
-	pairs := make([]string, len(cps))
-
-	for i, cp := range cps {
-		pairs[i] = currencyPairToBinanceTickerPair(cp)
-	}
-
-	return p.subscribePairs(pairs...)
-}
-
-// subscribeCandles subscribe to the candle channel for all currency pairs.
-func (p *BinanceProvider) subscribeCandles(cps ...types.CurrencyPair) error {
-	pairs := make([]string, len(cps))
-
-	for i, cp := range cps {
-		pairs[i] = currencyPairToBinanceCandlePair(cp)
-	}
-
-	return p.subscribePairs(pairs...)
-}
-
-// subscribedPairsToSlice returns the map of subscribed pairs as a slice.
-func (p *BinanceProvider) subscribedPairsToSlice() []types.CurrencyPair {
-	p.mtx.RLock()
-	defer p.mtx.RUnlock()
-
-	return types.MapPairsToSlice(p.subscribedPairs)
+	return nil // handled by the websocket controller
```

Comment:
Looks like `SubscribeCurrencyPairs` is a no-op on every provider now, should we remove it from the provider interface & stop using it altogether? 

Also, if we're keeping this in for reverse-compatibility, please update the function comments instead
```golang
// SubscribeCurrencyPairs currently performs a no-op; this is handled by the websocket controller.
func (p *BinanceProvider) SubscribeCurrencyPairs(cps 
```

Reply:
zeroed it https://github.com/umee-network/umee/pull/1554/commits/ab4d240ad9caf2dfe37ca00e5db36906d9016d88

Reply:
niceu. this will change the PF / peggo integration. 

cc: @RafilxTenfen